app/views/questions/_active_questions.html.erb

```diff
@@ -0,0 +1,27 @@
+<% if @active_questions.empty? %>
```

Comment:
Not for this PR (just a note no need for changes), but when this is connected with Competencies eventually when we created the Question_Indicator model, we probably need to change how this is formatted. Maybe show active questions for each Competency